_howdoi/how-do-i-launch-a-jupyter-notebook.adoc

```diff
@@ -0,0 +1,65 @@
+= launch a Jupyter notebook on OpenShift
+
+There are multiple ways to launch a Jupyter notebook on OpenShift with the
+radanalytics.io tooling. You can use the OpenShift console or the `oc` command
+line tool. Before running these instructions you will need to have installed
+the radanalytics.io manifest as referenced on the
+link:/get-started[Get Started] page.
+
+== Launching Jupyter from the OpenShift console
+
+To begin, select the "Jupyter Notebook" template from the "Add to Project"
+menu in your OpenShift console.
+
+pass:[<img src="/assets/howdoi/jupyter-launch-1.png" alt="Jupyter template" class="img-responsive">]
+
+After selecting the template and clicking past the information screen, you
+will be presented with several options for your notebook deployment. You will
+need to create a password for your notebook and optionally give it a name and
+the URI of a notebook file to download.
```

Comment:
nit:  should this say "IPython notebook file" so it's a little clearer to people who are new to Jupyter (and so that they're clear that they don't have one if they don't!)